pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/BaseClusterIntegrationTest.java

```diff
@@ -478,7 +479,8 @@ protected void stopKafka() {
    */
   protected long getCurrentCountStarResult()
       throws Exception {
-    return getPinotConnection().execute("SELECT COUNT(*) FROM " + getTableName()).getResultSet(0).getLong(0);
+    return getPinotConnection().execute(new Request("pql", "SELECT COUNT(*) FROM " + getTableName())).getResultSet(0)
```

Comment:
Why not make this "sql"? We will phase out pql anyway, so we can start doing this via sql all ths time?

Reply:
Ah ok, looks like this is deprecated.